### DISCUSSION TITLE
Instance creation using Terraform

### DISCUSSION THREAD

**Discussion by LavanyaRamesh319** [learndevopsonline/learndevopsonline#1875] [Q&A]
I am unable to see name in aws console when I try to create instance using Terraform. Instance is getting created but name is not displaying in console. 
![Screenshot_2024-03-30-06-38-48-711_com android chrome](https://github.com/learndevopsonline/learndevopsonline/assets/103929459/61dcd28b-56df-4b03-9d1a-c7c3072ece56)

**Comment by chaithanya1812** [comment]
Hi @LavanyaRamesh319 ,  In your code change below modification 
## change name ----> Name

**Comment by LavanyaRamesh319** [comment]
Its working now...... Thank you @chaithanya1812
